oaknational/Oak-Web-Application
# LESQ-934: New Content Banner Video

## Description

Music year: 2000

- Removes image from new content banner and replaces it with video

## Issue(s)

Fixes #LESQ-934

## How to test

1. Go to https://deploy-preview-2624--oak-web-application.netlify.thenational.academy
2. Click on the legacy option of any legacy subject that has new content (apart from Music)
3. You should see banner with video available to be played

## Screenshots

How it used to look (delete if n/a):
<img width="1026" alt="Screenshot 2024-07-19 at 10 45 17" src="https://github.com/user-attachments/assets/3f131e90-1e0d-4be5-a117-35ebd62eafa5">

How it should now look:


https://github.com/user-attachments/assets/ec6e2209-c13b-4050-a12b-f64f879ca194


## Checklist
